src/foolscap/schema.py

```diff
@@ -82,7 +84,10 @@
 DictOf = DictConstraint
 SetOf = SetConstraint
 
-
+if six.PY3:
+    unicode = str
```

Comment:
This seems a little weird; can we instead just get rid of `unicode()` use entirely..? e.g. by always prefixing strings with `u""` or `b""`?

Reply:
We can use "six.ensure_text(..)" etc for this. In this case I went for a quick shortcut really.